shared/teams/team/container.js

```diff
@@ -62,6 +62,10 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const onOpenFolder = () => dispatchProps._onOpenFolder(stateProps.name)
   const onManageChat = () => dispatchProps._onManageChat(stateProps.name)
   const onLeaveTeam = () => dispatchProps._onLeaveTeam(stateProps.name)
+  const yourType =
```

Comment:
Do you need the count check?

Reply:
Ah thanks, test on the next line made it unnecessary.